pkg/operator/operator/validations.go

```diff
@@ -452,6 +473,23 @@ func validatePredictor(predictor *userconfig.Predictor, projectFileMap map[strin
 		}
 	}
 
+	images := []string{
+		predictor.Image,
+		predictor.TFServeImage,
+	}
+	keys := []string{
+		userconfig.ImageKey,
+		userconfig.TFServeImageKey,
+	}
+	// do it in one go to avoid generating credentials twice
+	key, err := validateOverridenImages(images, keys)
```

Comment:
`validateOverridenImages()` should do the appropriate wrapping of images/keys, so here we can just return `err` without wrapping. After a long design discussion that we had a few months ago (which I'm happy to elaborate more on!), we decided that the "rule" is: assume that any information that's passed into the function is already included in the returned error message.

Reply:
> assume that any information that's passed into the function is already included in the returned error message

This! This is a very important bit. Thanks for giving me the perspective. And it makes sense now that I think of it. Consider it done.